consensus/service/src/api/client_api_service.rs

```diff
@@ -308,8 +367,21 @@ impl ConsensusClientApi for ClientApiService {
                     ConsensusGrpcError::NotServing.into()
                 }
             } else {
-                self.handle_proposed_tx(msg)
-                    .or_else(ConsensusGrpcError::into)
+                let result = self.handle_proposed_tx(msg);
+                // The block present below rate-limits suspicious behavior.
+                if let Err(_err) = &result {
```

Comment:
:wrench: Thinking `is_err()` would work here since the contents don't appear to be looked at
```suggestion
                if result.is_err() {
```